components/markdown/paragraph.tsx

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import { Box, BoxProps } from '@sparkpost/matchbox';
+import Container from './container';
 
 const Paragraph = (props: BoxProps): JSX.Element => {
-  return <Box mb="400">{props.children}</Box>;
+  return (
+    <Container>
+      <Box mb="550" as="p">
+        {props.children}
+      </Box>
```

Comment:
Same as I mentioned above, but the `<p>`s look to be 512px wide when design shows them at 600px wide.